### PR TITLE
Create files with class

### PR DESCRIPTION
**Pull request by MajStc**
Create:
- boardField.ts -> file with BoardField class 
- boardFieldWithRiddle.ts -> file with BoardFieldWithRiddle class that extends BoardField class

### CODE REVIEW

File: __tests__/boardFieldTDD.test.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    test('newField should have a getter of paragraphID',()=>{
+        expect(typeof (newField.getParagraphID)).toBe('string');
+    });
+
+    test('newField should have a setter for current status',()=>{
+        newField.setCurrentStatus = TestEnum.failed;
+        expect(newField.getStatus).toBe('FAILED');
+    });
+});
```
Comment (TenGosc007) [inline __tests__/boardFieldTDD.test.ts]: a blank line at the end of the file

File: __tests__/boardFieldWithPoints.test.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    test('Should have methods of normal BoardField class', ()=>{
+        expect(typeof(boardFieldWithAction.readParagraph)).toEqual('function');
+    })
+
+    test('Should have a method of adding a points to game state',()=>{
+        expect(typeof(boardFieldWithAction.addPoints)).toEqual('function');
+    })
+
+})
```
Comment (TenGosc007) [inline __tests__/boardFieldWithPoints.test.ts]: same as above

File: __tests__/boardFieldWithRiddleTDD.test.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    });
+
+    test('newFieldWithRiddle should have a method of reading a paragraph from normal BoardField',()=>{
+        expect(typeof (newFieldWithRiddle.readParagraph)).toBe('function');
+    });
+
+    test('newFieldWithRiddle should have a method of reading a riddle from BoardFieldWithRiddle',()=>{
+        expect(typeof (newFieldWithRiddle.readRiddle)).toBe('function');
+    });
+});
```
Comment (TenGosc007) [inline __tests__/boardFieldWithRiddleTDD.test.ts]: same as above

**Review by Veryyapeee — DISMISSED** [review]

File: src/boardField.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    constructor(
+        protected status: TestEnum, 
+        readonly fieldID: string, 
+        readonly paragraphID: string){}
+
+    get getStatus(){
+        return this.status;
+    }
+
+    get getFieldID(){
```
Comment (TenGosc007) [inline src/boardField.ts]: I suggest change names. For example protected _status and then get status()

File: src/boardField.ts [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    set setCurrentStatus(status: TestEnum){
+        this.status=TestEnum.failed;
+    }
+
+    readParagraph(){
+        // wywolanie funkcji z frontu do wczytania paragrafu
+    };
+}
```
Comment (TenGosc007) [inline src/boardField.ts]: same as above

File: src/boardFieldWithPoints.ts [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+/**
+ * @Class boardFieldWithPoints extend BoardField
+ *  extend normal BoardField by addPoints method, exploring this field grants user an additional action point
+ */
+export class BoardFieldWithPoints extends BoardField {
+    addPoints(){
+        // state.addpoints
+    }
+}
```
Comment (TenGosc007) [inline src/boardFieldWithPoints.ts]: same as above

File: src/boardFieldWithRiddle.ts [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+
+/**
+ * @class - boardFieldWithRiddle
+ *  extend normal BoardField class by a method that reads a riddle
+ */
+export class BoardFieldWithRiddle extends BoardField{
+    readRiddle(){
+        // wywolanie funkcji czytajacej zagadke
+    }
+}
```
Comment (TenGosc007) [inline src/boardFieldWithRiddle.ts]: same as above

**Review by TenGosc007 — APPROVED** [review]
LGTM!

**Review by Veryyapeee — APPROVED** [review]

**Pull request merged**